### DISCUSSION TITLE
miopen verbose output

### DISCUSSION THREAD

**Discussion by acxz** [rocm-arch/rocm-arch#970] [Support] [answered]
From @itsagoodbrain:

> I didn't want to bug report as this might be expected behavior, but when I'm using pytorch from community now I get an extremely verbose output from MIOpen. The following output is repeated thousands of time every time I evaluate the model.
> 
> ```
> MIOpen(HIP): Info [SetStream] stream: 0, device_id: 0
> MIOpen(HIP): Info [ConvolutionForward] algo = 3, workspace = 0
> MIOpen(HIP): Info [get_device_name] Raw device name: gfx1030
> ```
> 
> I tried exporting environmental variables to make sure the verbosity was set to debug only, but it had no effect. I'm assuming it's because those flags are for compiling not runtime. According to [these docs](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/DebugAndLogging.html) the default settings would only output `Info` logs if it was a debug build.

**Comment by tpkessler** [comment, marked as answer]
This is fixed with the latest version of miopen in the repos.
